runtime/shibuya/src/lib.rs

```diff
@@ -796,7 +796,7 @@ pub type Executive = frame_executive::Executive<
     frame_system::ChainContext<Runtime>,
     Runtime,
     AllPalletsWithSystem,
-    (DappsStakingMigrationV3,),
+    (DappsStakingMigrationV3, DappsStakingFixEraLength),
```

Comment:
We don't need V3 migration anymore - it might even mess something up.
Feel free to delete it.